library/src/main/java/com/owncloud/android/lib/common/operations/RemoteOperationResult.java

```diff
@@ -316,6 +320,10 @@ public RemoteOperationResult(boolean success, OkHttpMethodBase httpMethod) {
         this(success, httpMethod.getStatusCode(), httpMethod.getStatusText(), httpMethod.getResponseHeaders());
     }
 
+    public RemoteOperationResult(boolean success, OkHttpDavMethodBase davMethod) {
+        this(success, davMethod.statusCode());
+    }
```

Comment:
Again, it would be better to have `OkHttpMethodBase` and `OkHttpDavMethodBase` share a hierarchy or implement a common interface, so that `RemoteOperationResult` wouldn't need to care which of them it's dealing with.